docs/src/cookbooks/naming.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: docs
+title:  "Naming Cookbook"
+section: "chisel3"
+---
+
+
+### I still have _T signals, can this be fixed?
+
+First check - is the compiler plugin properly enabled? If so, these signals could be intermediate values which are
+consumed by either assertions or when predicates. In these cases, the compiler plugin often can't find a good prefix
+for the generated intermediate signals. We recommend you manually insert calls to `prefix` to fix these cases. We did
+this to RocketChip and saw huge benefits!
```

Comment:
```suggestion
this to Rocket Chip and saw huge benefits!
```

How do I check if the compiler plugin is enabled?

This might be a good place to tell people that Scalac plugins are enabled via the scalac option `-Xplugin:<path/to/jar>` and give an SBT command to check this, eg.
```
$ sbt
> show Compile / scalacOptions
```